### PR TITLE
LPS-198234 Create tests for DDMFormInstanceTableReferenceDefinition and DDMFormInstanceVersionTableReferenceDefinition

### PR DESCRIPTION
**Pull request by gislayne-vitorino**
This is an update for: [LPS-198234 ](https://liferay.atlassian.net/browse/LPS-198234)

### CODE REVIEW

**Comment by gislayne-vitorino** [comment]
ci:test:publications

**Comment by gislayne-vitorino** [comment]
ci:test:publications

**Comment by gislayne-vitorino** [comment]
ci:test:publications

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/change/tracking/spi/reference/DDMFormInstanceTableReferenceDefinition.java [hunk 1]
```diff
@@ -71,6 +72,12 @@ public void defineChildTableReferences(
 						DDMFormInstance.class.getName())
 				)
 			)
+		).singleColumnReference(
+			DDMFormInstanceTable.INSTANCE.structureId,
```
Comment (david-truong) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/change/tracking/spi/reference/DDMFormInstanceTableReferenceDefinition.java:76]: A column should only map to one other column.  You are mapping structureId twice.  The one to DDMStructureVersion is not needed

**Review by david-truong — CHANGES_REQUESTED** [review]

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Pull request closed without merging**